src/main/java/net/minecraftforge/event/entity/living/VillagerTradePopulationEvent.java

```diff
@@ -0,0 +1,64 @@
+package net.minecraftforge.event.entity.living;
+
+import java.util.List;
+import net.minecraft.entity.passive.EntityVillager;
+import net.minecraft.entity.passive.EntityVillager.ITradeList;
+import net.minecraftforge.fml.common.eventhandler.Cancelable;
+import net.minecraftforge.fml.common.eventhandler.Event;
+import net.minecraftforge.fml.common.eventhandler.Event.HasResult;
+import net.minecraftforge.fml.common.registry.VillagerRegistry.VillagerCareer;
+
+/**
+ * The VillagerTradePopulationEvent is fired when the trades are populated<br>
+ * in {@link EntityVillager#populateBuyingList()}.<br>
+ * It allows the modder to modify the trades of an individual Villager.<br>
+ * This event is fired via the {@link ForgeHooks#onVillagerTradePopulation}.<br>
```

Comment:
Don't put ForgeHooks, we know everything is fired from there.
Put where in vanilla was patched to call Forgehooks
This also helps us make sure it's properly updated every MC Version

Something like "This event is fired from ForgeHooks... via <vanilla method>"